index.js

```diff
@@ -1,4 +1,7 @@
+#!/usr/bin/env node
```

Comment:
Shebang added?

Reply:
Oops, forgot to take care of the root folder -- fixed!